app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -0,0 +1,50 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    # Generic JWT authenticator that receive JWT vendor configuration and uses to validate that the authentication
+    # request is valid, and return conjur authn token accordingly
+    Authenticate = CommandClass.new(
+      dependencies: {
+        token_factory: TokenFactory.new
+      },
+      inputs: %i[jwt_configuration authenticator_input]
+    ) do
+
+      def call
+        validate_and_decode_token
+        get_conjur_id
+        validate_restrictions
+        new_token
+      end
+
+      private
+
+      def validate_and_decode_token
+        unless @jwt_configuration.validate_and_decode_token
+          raise "validate and decode failed"
+        end
+      end
+
+      def get_conjur_id
```

Comment:
I don't understand this method.
It is called `get_conjur_id` but it doesn't return it.
Why is it called?

Reply:
https://github.com/cyberark/conjur/pull/2132#discussion_r621883469

Reply:
Changed function to 
```
def conjur_id
        # Will be changed when real get identity implemented
        @jwt_configuration.conjur_id
      end
```